src/leo-client-app/src/components/EditSchedules.tsx

```diff
@@ -8,26 +8,38 @@ import "../styles.css";
 import "./styles/Scheduler.css";
 import SatelliteName from "./SatelliteName";
 import { Box, Card, Typography, Stack, Button } from "@mui/material";
+import axios from "axios";
 
-type Props = {
-  noradId: string;
-};
-
-const EditScheduler = ({ noradId }: Props) => {
+const EditScheduler = () => {
   const queryClient = useQueryClient();
   const router = useRouter();
+  const { satId, scheduleId } = router.query as {
+    satId: string;
+    scheduleId: string;
+  };
   // const satelliteId = "655acd63d122507055d3d2ea";
   const satelliteId = router.query?.satelliteId?.toString() ?? "";
   // console.log(typeof(satelliteId), "hi");
   const adminUserId: string = "65a5e11fe0d601e0e8c4a385";
 
   // operator
   const userId: string = "65a8181f36ea10b4366e1dd9";
-  const scheduleId = "65a8182036ea10b4366e1de6";
   const isAdmin = adminUserId === userId;
```

Comment:
what about dynamic for user ids

Reply:
can be another pr, we only have 1 user for now anyways